src/main/java/com/example/App.java

```diff
@@ -5,6 +5,10 @@
 public class App {
     public App() throws IOException {
 //        super();
+        // add more comments
+        if (true) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://docker.for.mac.localhost:9000/coding_rules#rule_key=squid%3AS2589)